feature/gribi/otg_tests/route_removal_non_default_vrf_test/route_removal_non_default_vrf_test.go

```diff
@@ -0,0 +1,435 @@
+		t.Errorf("LossPct for flow got %v, want 0", got)
+	} else {
+		t.Log("Traffic can be forwarded between ATE port-1 and ATE port-2")
+	}
+
+	entry := verifyEntry(t, dut, nonDefaultVRF, ateDstNetEntryNonDefault)
+	if !entry {
+		t.Errorf("ipv4-entry/state/prefix does not contain entry, expected: %s", ateDstNetEntryNonDefault)
+	} else {
+		t.Logf("IP Entry for %s has NOT been removed from network instance: %s as confirmed from telemetry.", ateDstNetEntryNonDefault, nonDefaultVRF)
+	}
+
+	t.Log("Ensure that 203.0.113.0/24 (ateDstNetEntryDefault) has been removed by validating telemetry.")
+	entry = verifyEntry(t, dut, *deviations.DefaultNetworkInstance, ateDstNetEntryDefault)
+	if entry {
+		t.Errorf("ipv4-entry/state/prefix contains entry %s, expected no entry", ateDstNetEntryDefault)
+	} else {
+		t.Logf("IP Entry for %s has been successfully removed from network instance: %s as confirmed from telemetry.", ateDstNetEntryDefault, *deviations.DefaultNetworkInstance)
+	}
+}
+
+// configureDUT configures port1-2 on the DUT.
+func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
+	t.Helper()
+	d := gnmi.OC()
+
+	p1 := dut.Port(t, "port1")
+	p2 := dut.Port(t, "port2")
+
+	gnmi.Replace(t, dut, d.Interface(p1.Name()).Config(), dutPort1.NewOCInterface(p1.Name()))
+	gnmi.Replace(t, dut, d.Interface(p2.Name()).Config(), dutPort2.NewOCInterface(p2.Name()))
+
+}
+
+// configureATE configures port1, port2 on the ATE.
+func configureATE(t *testing.T, ate *ondatra.ATEDevice) gosnappi.Config {
+	t.Helper()
+	top := ate.OTG().NewConfig(t)
+
+	p1 := ate.Port(t, "port1")
+	p2 := ate.Port(t, "port2")
+
+	atePort1.AddToOTG(top, p1, &dutPort1)
+	atePort2.AddToOTG(top, p2, &dutPort2)
+
+	// Adding traffic flow
+	flowipv4 := top.Flows().Add().SetName("Flow")
+	flowipv4.Metrics().SetEnable(true)
+	e1 := flowipv4.Packet().Add().Ethernet()
+	e1.Src().SetValue(atePort1.MAC)
+	flowipv4.TxRx().Device().SetTxNames([]string{atePort1.Name + ".IPv4"}).SetRxNames([]string{atePort2.Name + ".IPv4"})
+	v4 := flowipv4.Packet().Add().Ipv4()
+	v4.Src().SetValue(atePort1.IPv4)
+	v4.Dst().Increment().SetStart("198.51.100.0").SetCount(255)
+
+	return top
+}
+
+// configureNetworkInstance configures network instance.
+func configureNetworkInstance(t *testing.T, dut *ondatra.DUTDevice) {
+
+	nonDefaultNI := networkInstance(t, nonDefaultVRF)
+	p1 := dut.Port(t, "port1")
+	niIntf := nonDefaultNI.GetOrCreateInterface(p1.Name())
+	niIntf.Subinterface = ygot.Uint32(0)
+	niIntf.Interface = ygot.String(p1.Name())
+
+	gnmi.Replace(t, dut, gnmi.OC().NetworkInstance(nonDefaultVRF).Config(), nonDefaultNI)
+}
+
+// networkInstance creates an OpenConfig network instance with the specified name
+func networkInstance(t *testing.T, name string) *oc.NetworkInstance {
+	d := &oc.Root{}
+	ni := d.GetOrCreateNetworkInstance(name)
+	ni.Description = ygot.String("Non Default routing instance created for testing")
+	ni.Type = oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L3VRF
+	ni.Enabled = ygot.Bool(true)
```

Comment:
I think you need to manually merge in some changes made to the ate_test version. In particular https://github.com/openconfig/featureprofiles/commit/b47bcfc517747439ab734566a2771acb2435eeec , which removes references to fields that no longer exist and prevent successful compilation.

Apologies for our delay in review that let this get out of sync.